app/audit/Utilities.scala

```diff
@@ -19,6 +19,8 @@ package audit
 import auth.MtdItUser
 import play.api.libs.json.{JsObject, Json}
 import utils.Utilities.JsonUtil
+import _root_.models.financialDetails.Charge
```

Comment:
Just a note: this automagically added `_root_` package is a bit ugly, usually devs try to get rid of it.

Reply:
Actually, I don't see an easy way to distinguish packages without renamings. Thus, let it be.